README.md

```diff
@@ -64,8 +64,8 @@ By default, this package does not bring in data from the Qualtrics Research Core
 
 ```yml
 vars:
-    qualtrics__using_core_contacts: True # default = False
-    qualtrics__using_core_mailing_lists: True # default = False
+    qualtrics__using_core_contacts: True # default = False 
+    qualtrics__using_core_mailing_lists: True # default = False 
```

Comment:
I noticed these variables are both attached to the same core API endpoint. What are your thoughts on consolidating these into one variable? Or does it make more sense to keep them separate?

Reply:
yeah i was thinking about this -- theoretically you could have core contacts but but not organize them into mailing lists, which is why i opted for two separate ones. 

i don't think that is a very common use case however (and this whole endpoint is not widely used to begin with), so perhaps the best path forward would be to start with one variable and split into two if a customer opens an issue?

Reply:
Thanks for the explanation! With this, I agree that having two separate variables is appropriate.